### ISSUE TITLE
[a11y]: Full Page Error, Decorative image has redundant alt text

### ISSUE THREAD

**Issue by anamikaanu96** [carbon-design-system/ibm-products#6232]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

v2.52.0-rc.0

### Automated testing tool and ruleset

Manual

### Assistive technology

JAWS

### Description

Reported by Raghu during QA Audit.

A decorative image on the page has been assigned alt text that conveys information already provided to the user through other means. 
Decorative images should have empty alt attributes (e.g., alt="") to indicate that they do not convey any meaningful content. 

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-full-page-error-fullpageerror--full-page-error&globals=viewport:basic

### Steps to reproduce

This was reported in all variants of the Full Page Error.

Open Full Page Error, check the decorative image.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
